maps/site53/items/clothing/solgov-head.dm

```diff
@@ -363,6 +363,8 @@
 	desc = "A dark red beret worn by members of the 'Red Right Hand' MTF unit, it feels kind of heavy for a beret."
 	icon_state = "alpha-beret"
 	item_state = "alpha-beret"
+	body_parts_covered = HEAD
+	armor = list(melee = 90, bullet = 90, laser = 90, energy = 95, bomb = 90, bio = 90, rad = 0)
```

Comment:
Make it as armored as an Alpha-1 helmet if those exist

Reply:
i don't think they exist